desc/protoparse/parser.go

```diff
@@ -600,8 +600,11 @@ func (r *parseResult) createFileDescriptor(filename string, file *fileNode) erro
 
 	isProto3 := false
 	if file.syntax != nil {
-		fd.Syntax = proto.String(file.syntax.syntax.val)
 		isProto3 = file.syntax.syntax.val == "proto3"
+		// https://github.com/protocolbuffers/protobuf/blob/0c3f43a6190b77f1f68b7425d1b7e1a8257a8d0c/src/google/protobuf/descriptor.cc#L2014
```

Comment:
Let's change the comment to `// proto2 is the default, so no need to set unless proto3`

Reply:
Done, although I might argue you want to keep this link around for the future, thoughts?